packages/dd-trace/src/exporters/common/request.js

```diff
@@ -106,11 +106,12 @@ function request (data, options, callback) {
         } catch {
           // ignore error
         }
+
         const responseData = buffer.toString()
         if (responseData) {
           errorMessage += ` Response from the endpoint: "${responseData}"`
         }
-        const error = new Error(errorMessage)
+        const error = new log.NoTransmitError(errorMessage)
```

Comment:
This prevents us from sending the HTTP response via telemetry which is where PII could slip in.